usb/hidapi.go

```diff
@@ -44,6 +45,7 @@ func (b *HIDAPI) Enumerate() ([]core.USBInfo, error) {
 	b.mw.Log("low level done")
 
 	for _, dev := range devs { // enumerate all devices
+		dev := dev
```

Comment:
we take pointer of something we range through... gosec shows it as an issue even when we don't use it in a different goroutine, but still, probably should be like this, for the future etc